js/pages/concept-sets/components/tabs/conceptset-expression.js

```diff
@@ -48,14 +46,30 @@ define([
       );
     }
 
+    toggleCheckbox(d, field) {
+      if (this.canEdit) {
+        const concept = this.selectedConcepts().find(i => !!i.concept && !!d.concept && i.concept.CONCEPT_ID === d.concept.CONCEPT_ID);
```

Comment:
Seems like this repeats code in `js/components/conceptset/conceptset-editor.js`: wouldn't it be better to move the code into a shared function?

Reply:
Moved toggleCheckbox and renderCheckbox to CommonUtils: https://github.com/OHDSI/Atlas/pull/1852/commits/c9cd4b96dbe7da048229f1d782c9efb6c2173e3e